tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -58,6 +58,41 @@ public static IEnumerable<object[]> ProbeTests()
         return DebuggerTestHelper.AllProbeTestTypes();
     }
 
+    [SkippableFact]
+    [Trait("Category", "EndToEnd")]
+    [Trait("RunOnWindows", "True")]
+    public async Task TransparentCodeCtorInstrumentationTest()
```

Comment:
I'm missing something obvious here - why can't this just be part of the regular probe tests?

Reply:
Two reasons:
1. I can't place the `MethodProbeTestDataAttribute` on SecurityTransparentTest's c'tor as it's the default implementation given by the compiler.
2. `SecurityTransparentTest` type exists in another assembly and in the regular probe tests we scan the class of the `IRun` itself for `MethodProbeTestDataAttribute` - there is no easy way to do so to every single method that is executing during the test.

Reply:
Got it. Fair enough, thanks.